spark/src/main/scala/org/apache/spark/sql/delta/TableFeature.scala

```diff
@@ -751,7 +751,7 @@ private[sql] object TestRemovableWriterFeature
   }
 
   override def validateRemoval(snapshot: Snapshot): Boolean =
-    !snapshot.metadata.configuration.contains(TABLE_PROP_KEY)
+    !snapshot.metadata.configuration.get(TABLE_PROP_KEY).exists(_.toBoolean)
```

Comment:
Can you please add a comment explaining what is this doing?
`Make sure that the property is not enabled on the table.`
Or alternatively `snapshot.metadata.configuration.get(TABLE_PROP_KEY).forall(!_.toBoolean)` is a bit more readable.